waveform_editor/gui/editor.py

```diff
@@ -85,22 +89,32 @@ def on_value_change(self, event):
         annotations = waveform.annotations
         self.code_editor.annotations = list(annotations)
         if self.config.parser.parse_errors:  # Handle errors
-            self.error_alert.object = (
-                "### The YAML did not parse correctly\n  "
-                f"{self.config.parser.parse_errors[0]}"
+            self.create_error_alert(
+                "The YAML did not parse correctly\n  "
+                + f"{self.config.parser.parse_errors[0]}",
+                "danger",
             )
-            self.error_alert.alert_type = "danger"
         else:  # No errors
             if self.code_editor.annotations:  # Handle warnings
-                self.error_alert.object = (
-                    f"### There was an error in the YAML configuration\n{annotations}"
+                self.create_error_alert(
+                    "There was an error in the YAML configuration\n" + f"{annotations}",
+                    "warning",
                 )
-                self.error_alert.alert_type = "warning"
             else:
                 self.error_alert.object = ""  # Clear any previous errors or warnings
             # There are no errors: update self.waveform
             self.waveform = waveform
 
+    def create_error_alert(self, message, alert_type="danger"):
```

Comment:
Let's remove the default argument?
```suggestion
    def create_error_alert(self, message, alert_type):
```